# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1302768388 -7200
#      Thu Apr 14 10:06:28 2011 +0200
# Node ID 46cbccfb46c1a41127e6418719cc17591d83d935
# Parent  79e2f4d8fa7f88bb60878a9d60e3e1b09c141cf8
Fixed bug with displaying the terminal size while resizing.

diff --git a/gui//src/TerminalDisplay.cpp b/gui//src/TerminalDisplay.cpp
--- a/gui//src/TerminalDisplay.cpp
+++ b/gui//src/TerminalDisplay.cpp
@@ -1133,17 +1133,17 @@ void TerminalDisplay::showResizeNotifica
         _resizeWidget->setAlignment(Qt::AlignCenter);
 
         _resizeWidget->setStyleSheet("background-color:palette(window);border-style:solid;border-width:1px;border-color:palette(dark)");
 
         _resizeTimer = new QTimer(this);
         _resizeTimer->setSingleShot(true);
         connect(_resizeTimer, SIGNAL(timeout()), _resizeWidget, SLOT(hide()));
      }
-     QString sizeStr = i18n("Size: %1 x %2", _columns, _lines);
+     QString sizeStr = QString("Size: %1 x %2").arg(_columns).arg(_lines);
      _resizeWidget->setText(sizeStr);
      _resizeWidget->move((width()-_resizeWidget->width())/2,
                          (height()-_resizeWidget->height())/2+20);
      _resizeWidget->show();
      _resizeTimer->start(1000);
   }
 }
 
